### DISCUSSION TITLE
entrypoint,d scripts execution permissions problem

### DISCUSSION THREAD

**Discussion by santiky** [Tecnativa/doodba#634] [Doodba - The image]
Hello,

i'm having troubles trying to insert a script in odoo/custom/entrypoint.d folder. I have added the script in folder with execution permissions but when is copied to the container, the execution bit is removed and it is not executed when container boots up. I have resolved this adding a new line in odoo/dockerfile chmoding the file in container but it looks like a workaround... am i loosing some point or is it a doodba issue?

I'm using copier template in ubuntu 24.04.

**Comment by ap-wtioit** [comment]
It's working on our side (also on Ubuntu 24.04 and testing with 22.04 and 20.04 as well), and in the [tests](https://github.com/Tecnativa/doodba/blob/master/tests/scaffoldings/entrypoint/custom/entrypoint.d/60-customize_entrypoint.py)

Which docker version are you using?

**Reply by santiky** [reply]
I'm using stock docker in ubuntu apt repositories Docker version 24.0.7, build 24.0.7-0ubuntu4.1
but it was my bad, i was not declaring right the env python header in script, thank you very much for the tests link, it has been very helpful.
